### ISSUE TITLE
Warning while using hexbin filter

### ISSUE THREAD

**Issue by pierotofy** [PDAL/PDAL#2557]
Test case:

```
pdal translate -i input.laz -o out.json --metadata meta.json hexbin --filters.hexbin.edge_length=1
```

Returns in the console: `(pdal info filters.hexbin Error) GDAL failure (1) OGR_G_RemoveGeometry() not supported on polygons yet.`

`out.json` and `meta.json` are still populated (correctly?) so this looks like a warning. Thought I'll report it in case it's the cause of some more severe problem.

Removing `--filters.hexbin.edge_length=1` makes the warning disappear.

**Comment by abellgithub** [comment]
Sorry about this.  This is something that got fixed in GDAL 2.3.  I don't think we're going to fix it -- in the future we'll require a newer GDAL release.

You might see some small interior polygons that would otherwise be removed still existing, but that's the extent of the issue.

**Comment by pierotofy** [comment]
Thanks for looking into it! Will wait for a GDAL release.
